components/server/src/container-module.ts

```diff
@@ -373,4 +375,5 @@ export const productionContainerModule = new ContainerModule((bind, unbind, isBo
 
     bind(RedisClient).toSelf().inSingletonScope();
     bind(RedisMutex).toSelf().inSingletonScope();
+    rebind(DataCache).to(DataCacheRedis).inSingletonScope();
```

Comment:
:thought_balloon: If we want to make this a transparent, data-level cache: What about making this an (optional) parameter to `dbContainerModule(cache: DataCache = DataCache.NoOp)`?
If feel it's a bit more direct and harder to overlook, especially in test cases. :nail_care: